content/switch-from-macos-to-popos.md

```diff
@@ -117,7 +117,15 @@ Both macOS and Pop!\_OS use workspaces to organize open application windows
 
 macOS uses multi-touch gestures, or CTRL+[up arrow] or [down arrow], to launch "Mission Control" and CTRL+[left arrow] or [right arrow] are used to navigate between workspaces.
 
-In Pop!\_OS users press the SUPER key to enter "Overview" and show all workspaces. They can also use CTRL+SUPER+[up arrow] or [down arrow] to navigate between workspaces.
+#### Accessing Overview
+
+In Pop!\_OS the "Overview" will show all workspaces and the key(s) to access it is different on certain versions.  
+
+| Pop!\_OS 20.04+ | Pop!\_OS 22.04+ |
```

Comment:
"20.04+" implies that version and all versions above, but this isn't really true for 22.04. Also, it may be worth mentioning that these are defaults, but you can change SUPER key behavior in Settings > Desktop > Desktop Options. I might suggest formatting the table like this: 

| Pop!\_OS Version | Display Overview Default |
|------------|----------------------------------|
| 20.04 LTS | <kbd>SUPER</kbd> |
| 21.04 | <kbd>SUPER</kbd> |
| 21.10 | <kbd>SUPER</kbd> |
| 22.04 LTS | <kbd>SUPER</kbd>+<kbd>D</kbd> |

>NOTE: <kbd>SUPER</kbd> key behavior can be modified in Settings > Desktop > Desktop Settings.